contracts/JBTiered721Delegate.sol

```diff
@@ -38,6 +38,11 @@ contract JBTiered721Delegate is JB721Delegate, Ownable, IJBTiered721Delegate, IE
   error RESERVED_TOKEN_MINTING_PAUSED();
   error TRANSFERS_PAUSED();
 
+  //*********************************************************************//
+  // ------------------------- public constants ------------------------ //
+  //*********************************************************************//
+  uint256 public constant MAX_ROYALTY_RATE = 200;
```

Comment:
should we keep `MAX_ROYALTY_RATE` as `10_000` like `max_reserved_rate` for consistency ?

Reply:
no because it has to fit in a uint8, pretty sure.

Reply:
ill play with storage once all open PRs have been merged